src/main/java/core/basesyntax/producers/FigureProducer.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.producers;
+
+import core.basesyntax.Figure;
+import core.basesyntax.shape.Circle;
+import core.basesyntax.shape.IsoscelesTrapezoid;
+import core.basesyntax.shape.Rectangle;
+import core.basesyntax.shape.RightTriangle;
+import core.basesyntax.shape.Square;
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```

Comment:
```suggestion
    public static final int COUNT_OF_FIGURE = 5;
```
Figure is a class, we don't implement classes only interfaces

Reply:
done